### PR TITLE
[a11y] com joomla update

### PR DESCRIPTION
**Pull request by brianteeman**
Continuing the work to prevent assistive technology reading out the value of an icon. This PR addresses the icons in com_joomlaupdate as shown in the screenshots below.

screenshotr11-23-52
screenshotr11-30-55

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9fabb6e7462f8179d26a988a6915b665ac8e357b<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15067">issues.joomla.org/tracker/joomla-cms/15067</a>.</sub>

File: administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php [hunk 1]
```diff
@@ -59,14 +59,14 @@
...
-							<span class="icon-star hasTooltip" title="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>"></span>
+							<span class="icon-star hasTooltip" title="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>" aria-hidden="true"></span>
 							<label for="mod-login-secretkey" class="element-invisible">
 								<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>
 							</label>
 						</span>
 						<input name="secretkey" autocomplete="off" tabindex="3" id="mod-login-secretkey" type="text" class="input-medium" placeholder="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>" size="15"/>
 						<span class="btn width-auto hasTooltip" title="<?php echo JText::_('JGLOBAL_SECRETKEY_HELP'); ?>">
-							<span class="icon-help"></span>
+							<span class="icon-help" aria-hidden="true"></span>
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php]: I think we need to add `<span class="element-invisible">....</span>` (psuedo-code).

File: administrator/components/com_joomlaupdate/views/update/tmpl/finaliseconfirm.php [hunk 2]
```diff
@@ -60,14 +60,14 @@
...
-							<span class="icon-star hasTooltip" title="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>"></span>
+							<span class="icon-star hasTooltip" title="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>" aria-hidden="true"></span>
 							<label for="mod-login-secretkey" class="element-invisible">
 								<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>
 							</label>
 						</span>
 						<input name="secretkey" autocomplete="off" tabindex="3" id="mod-login-secretkey" type="text" class="input-medium" placeholder="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>" size="15"/>
 						<span class="btn width-auto hasTooltip" title="<?php echo JText::_('JGLOBAL_SECRETKEY_HELP'); ?>">
-							<span class="icon-help"></span>
+							<span class="icon-help" aria-hidden="true"></span>
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/views/update/tmpl/finaliseconfirm.php]: I think we need to add `<span class="element-invisible">....</span>` (psuedo-code).

File: administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php (hunk 1 above)
Comment (brianteeman) [inline administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php]: I dont think so as the button already has a title

File: administrator/components/com_joomlaupdate/views/update/tmpl/finaliseconfirm.php (hunk 2 above)
Comment (brianteeman) [inline administrator/components/com_joomlaupdate/views/update/tmpl/finaliseconfirm.php]: I dont think so as the button already has a title

File: administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php (hunk 1 above)
Comment (Quy) [inline administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php]: https://www.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/

I don't know if this information is current or not.
> If you want to hide content from mobile and tablet users as well as assistive tech users and keyboard only users, use the title attribute.

File: administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php [hunk 3]
```diff
@@ -45,7 +45,7 @@
 			<div class="controls">
 				<div class="input-prepend input-append">
 					<span class="add-on">
-						<span class="icon-lock hasTooltip" title="<?php echo JText::_('JGLOBAL_PASSWORD'); ?>"></span>
+						<span class="icon-lock hasTooltip" title="<?php echo JText::_('JGLOBAL_PASSWORD'); ?> aria-hidden="true""></span>
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php]: Move ` aria-hidden="true"` before `></span>`.

File: administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php [hunk 4]
```diff
@@ -45,7 +45,7 @@
 			<div class="controls">
 				<div class="input-prepend input-append">
 					<span class="add-on">
-						<span class="icon-lock hasTooltip" title="<?php echo JText::_('JGLOBAL_PASSWORD'); ?> aria-hidden="true""></span>
+						<span class="icon-lock hasTooltip" title="<?php echo JText::_('JGLOBAL_PASSWORD'); ?> aria-hidden="true"></span>
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/views/upload/tmpl/captive.php]: Not typo. Add `"` after `JText::_('JGLOBAL_PASSWORD'); ?>`.

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 9023a5222d885b9327e0408cb1f99fb88316af33<br /><br />Done code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15067">issues.joomla.org/tracker/joomla-cms/15067</a>.</sub>

**Comment by brianteeman** [comment]
@franz-wohlkoenig this has two good tests - the count was reset due to updating the branch - nothing changed in the PR<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15067">issues.joomla.org/tracker/joomla-cms/15067</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests. Thanks for Hint @brianteeman

**Comment by aminfke** [comment]
screen shot 2017-05-04 at 04 09 43

i have problem like this after i replace the language.php!!

anybody can help me???<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15067">issues.joomla.org/tracker/joomla-cms/15067</a>.</sub>

**Comment by aminfke** [comment]
i just refer this forum https://issues.joomla.org/tracker/joomla-cms/15587!!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15067">issues.joomla.org/tracker/joomla-cms/15067</a>.</sub>

**Comment by brianteeman** [comment]
@aminfke this is nothing to do with this PR

**Comment by brianteeman** [comment]
thanks

**Pull request merged**